v2/concordium/types.proto

```diff
@@ -499,6 +530,14 @@ message AccountInfo {
   // Canonical address of the account. This is derived from the first credential
   // that created the account.
   AccountAddress address = 10;
+  // The stake on the account that is in cooldown.
+  // There can be multiple amounts in cooldown that expire at different times.
+  repeated Cooldown cooldowns = 11;
+  // The available (unencrypted) balance of the account (i.e. that can be transferred
+  // or used to pay for transactions). This is the balance minus the locked amount.
+  // The locked amount is the maximum of the amount in the release schedule and
+  // the total amount that is actively staked or in cooldown (inactive stake).
+  Amount available_balance = 12;
```

Comment:
**Suggestion (non-blocking):** Like above, could we mention that this field was introduced in P7 and how it behaves for protocols prior to that